src/Integrations/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -288,6 +311,125 @@ function ($exceptionHandler, $scope, $args) use ($rootSpan, $integration) {
             }
         );
 
+        // Used by Laravel >= 5.0
+        \DDTrace\hook_method(
+            'Illuminate\Auth\SessionGuard',
+            'attempt',
+            null,
+            function ($This, $scope, $args, $loginSuccess) use ($rootSpan, $integration) {
+                if ($loginSuccess) {
```

Comment:
Is there enough information to call `track_user_login_success_event` here or is there an advantage to having a separate hook for it?